delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/syntax/UriSyntax.scala

```diff
@@ -41,6 +43,16 @@ final class UriOps(private val uri: Uri) extends AnyVal {
     */
   def toIri: Iri = Iri.unsafe(uri.toString)
 
+  /**
+   * Adds a segment to the end of the Uri
+   */
+  def /(segment: Label): Uri = /(segment.value)
+
+  /**
+   * Add a final slash to the uri
+   */
+  def finalSlash() : Uri = uri.copy(path = uri.path ++ Path./)
```

Comment:
You probably want to first check that the current uri does not already end with slash (`uriString2.path.endsWithSlash`)